jsp/src/main/java/com/sun/ts/tests/jsp/api/jakarta_servlet/jsp/tagext/tagvariableinfo/URLClientIT.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (c) 2007, 2023 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+/*
+ * $Id$
+ */
+
+/*
+ * @(#)URLClient.java	1.2 10/09/02
+ */
+
+package com.sun.ts.tests.jsp.api.jakarta_servlet.jsp.tagext.tagvariableinfo;
+
+
+import com.sun.ts.tests.jsp.common.client.AbstractUrlClient;
+
+/**
+ * Test client for TagVariableInfo. Implementation note, all tests are performed
+ * within a TagExtraInfo class. If the test fails, a translation error will be
+ * generated and a ValidationMessage array will be returned.
+ */
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit5.ArquillianExtension;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.jboss.shrinkwrap.api.asset.UrlAsset;
+
+import java.lang.System.Logger;
+
+@ExtendWith(ArquillianExtension.class)
+public class URLClientIT extends AbstractUrlClient {
+
+  private static final Logger logger = System.getLogger(URLClientIT.class.getName());
+
+  @BeforeEach
+  void logStartTest(TestInfo testInfo) {
+    logger.log(Logger.Level.INFO, "STARTING TEST : "+testInfo.getDisplayName());
+  }
+
+  @AfterEach
+  void logFinishTest(TestInfo testInfo) {
+    logger.log(Logger.Level.INFO, "FINISHED TEST : "+testInfo.getDisplayName());
+  }
+
+
+
+  public URLClientIT() throws Exception {
+    setup();
+    setContextRoot("/jsp_tagvarinfo_web");
+
+    }
+
+  @Deployment(testable = false)
+  public static WebArchive createDeployment() throws IOException {
+
+    String packagePath = URLClientIT.class.getPackageName().replace(".", "/");
+    WebArchive archive = ShrinkWrap.create(WebArchive.class, "jsp_tagvarinfo_web.war");
+    archive.addClasses(TagVarInfoTEI.class,
+            com.sun.ts.tests.jsp.common.util.JspTestUtil.class,
+            com.sun.ts.tests.jsp.common.util.BaseTCKExtraInfo.class,
```

Comment:
Maybe just say `JspTestUtil.class` instead of `com.sun.ts.tests.jsp.common.util.JspTestUtil.class`? 

Does the reader of the code really need to know the package JspTestUtil comes from at that location?